views/index.html

```diff
@@ -95,31 +95,29 @@ <h1 class="animated fadeInUp">CVR</h1>
 <code>
 #!/bin/bash
 
-# Required Travis config:
-# - Add the repo token in the variable CVR_TOKEN as a secure property in the Travis configuration
```

Comment:
Given that the script doesn't use the token at all, the text above the script (outside this diff) doesn't entirely make sense anymore.